packages/composites/circuit-breaker/README.md

```diff
@@ -0,0 +1,49 @@
+# Chainlink Circuit-breaker Composite Adapter
+
+This composite adapter makes use of 2 EA sources to capture pair values
+
+## Configuration
+
+The adapter takes the following environment variables:
+
+| Required? |          Name          |                 Description                 | Options | Defaults to |
+| :-------: | :--------------------: | :-----------------------------------------: | :-----: | :---------: |
+|           | `[source]_ADAPTER_URL` | The adapter URL to query for any `[source]` |         |             |
+
+## Running
+
+See the [Composite Adapter README](../README.md) for more information on how to get started.
+
+### Input Params
+
+| Required? |                 Name                 |                      Description                       |
+| :-------: | :----------------------------------: | :----------------------------------------------------: |
+|    ✅     |              `sources`               | An array or comma delimited 2 source adapters to query |
```

Comment:
Can we split this into `primarySource` and `backup/secondarySource`?
Since we only support two sources anyways, I think this would help in being more explicit

Reply:
done